src/corehost/cli/deps_resolver.cpp

```diff
@@ -164,6 +164,18 @@ void deps_resolver_t::setup_shared_store_probes(
     }
 }
 
+pal::string_t deps_resolver_t::get_probe_directories()
+{
+    pal::string_t directories;
+    for (const auto& pc : m_probes)
```

Comment:
When users assume the layout of the folders and they dont match expectation. this API is about exposing the paths that were specified with --additionalprobingpath or are implicit version of it (e.g. store location). This is not about servicing or FX locations. We should ensure the API places the user in the pit of success - not additional complexity.

What are your concerns in fixing the host to account for this, as this was expected to be in place to ensure the change is correct?